Engineering-Reference/GoogleDestinationAO.md

```diff
@@ -389,55 +373,55 @@ Default Value: File
 
 Select how file permissions will be applied when migrating to Shared Drive.
 
-Change Conditions: ??
+Change Conditions: None - No file permissions migrated. File - Original file permissions migrated. Root - Any items on file ACLs will be added as members of the Shared Drive.
 
 ### Shared Drive Same Domain Migration Type <a name="shasammigt"></a>
 {: .no_toc }
 Default Value: Move
 
 When migrating a Google Drive to Shared Drive within the same domain, move or copy the files and folders.
 
-Change Conditions: ??
+Change Conditions: Move will move the items to the target Shared Drive and Copy will copy them while preserving the original.
 
 ### Shared Drive Default Managers <a name="shdrideman"></a>
 {: .no_toc }
 Default Value: None
 
 Specify the list of managers email addresses. Should be valid user emails. Group emails are not allowed. These will be applied temporarily during migration of the Shared Drive, in order to improve performance, and removed upon completion if 'Remove Shared Drive Default Managers' is set to true.
 
-Change Conditions: ??
+Change Conditions: A list of default managers can be added.
 
 ### Migrate External Shared Drive Members <a name="migexshdrmem"></a>
 {: .no_toc }
 Default Value: Off
 
 Migrate External Shared Drive members when migrating a Shared Drive to Shared Drive. Please note sharing with external users will also be migrated.
 
-Change Conditions: ??
+Change Conditions: When enabled external Shared Drive members will be migrated.
 
 ### Shared Drive Folder Permissions <a name="shdrfolper"></a>
 {: .no_toc }
 Default Value: None
 
 Select how folder permissions will be applied when migrating to Shared Drive.
 
-Change Conditions: ??
+Change Conditions: None - No folder permissions migrated. Folder - Original folder permissions migrated. Root - Any items on folder ACLs will be added as members of the Shared Drive.
```

Comment:
Huh? how can you have no file permissions migrated but original file permissions migrated?

Reply:
None, File and Root are the three options. Will make it clearer.